src/plugins/timeConductor/independent/independentTimeConductorPopUpManager.js

```diff
@@ -31,14 +31,17 @@ export default {
       conductorPopup: null
     };
   },
+  beforeUnmount() {
+    if (this.conductorPopup.parentNode === document.body) {
```

Comment:
I think we just need a null check here

Reply:
I don't think so. We would never be in a state where `this.conductorPopUp` would be null when we get to this code.

Reply:
If you navigate away from a displayLayout with an Overlay Plot in it without opening the Overlay Plot's ITC you will get into that state and it crashes navigation, see my review comment